### ISSUE TITLE
Tabbing sticks on the gear button

### ISSUE THREAD

**Issue by rufataliy** [hackforla/311-data#903]
### Dependency
We are removing the gear from this layout.  Once that has been completed we can review this issue to see if tabing is still broken and update the steps to reproduce accordingly

### Overview
We need the site to work for screen readers you should be able t tab though all the navigation elements on the page from the top menu through the entire map nav

#### Describe the bug

While tabbing, focus sticks on the gear button and doesn't move forward to the next element.

<details>
<summary>To Reproduce</summary>

1. run the application
2. Start tabbing
3. After nav tab moves to the gear button
4. Press enter => a sidebar will appear
5. Press enter again => sidebar will close
6. Press tab => should switch to the dropdown button. 
7. **Error** => tab is still on gear the button
</details>

<details>
<summary>Expected behavior</summary>

Tab should switch to the dropdown button
</details>

**This was tested using Desktop:**
 - OS: Win10
 - Browser Chrome
 - Version 87

#### Links you will need 
- [Figma](https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI)
- Dev Site: https://dev.311-data.org/
 
<details>
<summary>Image of design before and after we remove the gear (after on the left)</summary>

![Menu Redesign](https://user-images.githubusercontent.com/20293568/147433067-1e6962ec-a1aa-4d8d-a6e4-821bb99c00d7.png)
